app/components/session/home/home.controller.js

```diff
@@ -1,19 +1,54 @@
-phinisiApp.controller('detailsController', function($scope, $http){
-	$scope.storeDetails = {
-		store_name: '',
-		store_logo: '',
-		phone_number: '',
-		address: '',
-		province: 'Select your province...',
-		city: 'Select your city...'
-	}
+phinisiApp.controller('detailsController', ['$scope', '$http', '$window', '$log', 
+	function($scope, $http, $window, $log){
+		$scope.storeDetails = {
+			store_name: '',
+			store_logo: '',
+			phone_number: '',
+			address: '',
+			province: '',
+			city: ''
+		}
+		$scope.provinces = [{id:1, name:'Jawa Barat'}, {id:2, name: 'Banten'}];
+		$scope.cities = [{provinceId: 1, cityId: 1, name:'Bandung'}, {provinceId: 1, cityId: 2, name:'Bogor'}, {provinceId: 2, cityId: 3, name:'Tangerang'},];
+		$scope.$log = $log;
 
-	$scope.provinces = [ 'Jawa Barat', 'Jakarta', 'Sumatera Utara' ];
-
-	$scope.city = [ 'tes' ]
-
-	$scope.submit = function () {
-		
-	}
-
-})
+		$scope.submit = function(){
+				//get store details from model
+				console.log($window.sessionStorage.token);
+				console.log($scope.storeDetails);
+				$http.post(
+					//url
+					phinisiEndpoint + '/merchant/info/create',
+					//data
+					{
+					  token: $window.sessionStorage.token, 
```

Comment:
you don't need to put token here. I have a function at app.auth.js that will intercept all outgoing request and put token manually there. So you can delete this line